iceoryx_posh/test/moduletests/test_popo_condition_variable.cpp

```diff
@@ -140,3 +140,23 @@ TEST_F(ConditionVariable_test, NotifyWhileWaitingResultsNoTimeoutMultiThreaded)
     m_signaler.notifyOne();
     waiter.join();
 }
+
+TEST_F(ConditionVariable_test, MoveConditionVariableSignalerIsSuccessful)
+{
+    /// @todo
+}
+
+TEST_F(ConditionVariable_test, MoveConditionVariableWaiterIsSuccessful)
+{
+    /// @todo
+}
+
+TEST_F(ConditionVariable_test, MoveAssignConditionVariableSignalerIsSuccessful)
+{
+    /// @todo
+}
+
+TEST_F(ConditionVariable_test, MoveAssignConditionVariableWaiterIsSuccessful)
+{
+    /// @todo
+}
```

Comment:
@elBoberido @elfenpiff When is a good timeslot to implement move and move assign for all building blocks? I'd say shortly before going live with #25

Reply:
Are they currently deleted? If yes, then this test should be disabled with a `/// @note move c'tor currently deleted` or completely removed.
@elfenpiff and I had an idea how to easily check for compile failures, e.g. if you want to check if the move c'tor is really deleted. I should probably write an issue for that

Reply:
Yep, they are currently deleted. We decided to implement them as one of the last points of #25, hence a compile failure check is not needed here. Added the comment and disabled them.

Reply:
Well, we do not have something like this for other classes were we deleted the move c'tor and assignment operator. Not sure if it should just be deleted. We should maybe also create an issue to create compile failure tests for all classes which have deleted c'tors/assignment operators

Reply:
Sure, created #222